sdk/appconfiguration/app-configuration/src/appConfigCredential.ts

```diff
@@ -19,12 +20,14 @@ export function appConfigKeyCredentialPolicy(credential: string, secret: string)
     async sendRequest(request: PipelineRequest, next: SendRequest): Promise<PipelineResponse> {
       const verb = request.method;
       const utcNow = new Date().toUTCString();
+      logger.info("[appConfigKeyCredentialPolicy] Generating a SHA-256 hash");
```

Comment:
NIT

Can the message include "using the body" perhaps?
Maybe "Computing SHA-256 hash from the request body"?